rust/agama-server/src/l10n/model/locale.rs

```diff
@@ -110,7 +112,20 @@ impl LocalesDatabase {
                 id: code.clone(),
                 language: language_label,
                 territory: territory_label,
+                consolefont: language
+                    .consolefonts
+                    .consolefont
+                    .first()
+                    .unwrap_or(&RankedConsoleFont {
```

Comment:
You are using an artificial value (with id `""`) to model the fact that it does not have a console font. IMHO, you should leverage the `Option<T>` type here.